<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EMS SCADA</string>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>맑은 고딕</family>
    </font>
   </property>
   <widget class="QFrame" name="totalframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="headframe">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(41, 42, 55);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="iconframe">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>7</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/iot);
background-repeat: no-repeat;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblTitle">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>7</y>
        <width>481</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>EMS IoT Dashboard</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="greetingFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>220</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 90);
border-style: outset;
border-width : 5px;
border-color: rgb(0, 0, 255);
border-radius : 10;



</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="weatherFrame">
      <property name="geometry">
       <rect>
        <x>78</x>
        <y>160</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/sun);
background-repeat : none;
border : none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblDate">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border: none;</string>
      </property>
      <property name="text">
       <string>9999-99-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblDay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>64</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border: none;</string>
      </property>
      <property name="text">
       <string>Wednesday</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTime">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>109</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border: none;</string>
      </property>
      <property name="text">
       <string>23 : 55</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblGreeting">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>230</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(210, 210, 210);
border: none;</string>
      </property>
      <property name="text">
       <string>Good Afternoon!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="tempFrame">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>271</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(71, 0, 0);
border-style: outset;
border-width : 5px;
border-color: rgb(180, 50, 0);
border-radius : 10;



</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="tempIconFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/temp);
background-repeat : none;
border : none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: none;</string>
      </property>
      <property name="text">
       <string>EMS99 Temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 25);
border: none;</string>
      </property>
      <property name="text">
       <string>99.9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrTemp_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 25);
border: none;</string>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="humidFrame">
     <property name="geometry">
      <rect>
       <x>521</x>
       <y>90</y>
       <width>271</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 50, 115);
border-style: outset;
border-width : 5px;
border-color: rgb(0, 50, 180);
border-radius : 10;



</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="tempIconFrame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/humid);
background-repeat : none;
border : none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblHumidTitle">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border: none;</string>
      </property>
      <property name="text">
       <string>EMS99 Humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrHumidity">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(190, 233, 255);
border: none;</string>
      </property>
      <property name="text">
       <string>99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrHumidity2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(190, 233, 255);
border: none;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dial_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="dashboard.qrc"/>
 </resources>
 <connections/>
</ui>
